source/ui/dataman/AMScanView.cpp

```diff
@@ -689,48 +763,26 @@ AMScanViewExclusiveView::AMScanViewExclusiveView(AMScanView* masterView) : AMSca
 	gl->setSizePolicy(sizePolicy());
 	setLayout(gl);
 
-
 	// the list of plotItems_ needs one element for each scan.
+
 	for(int scanIndex=0; scanIndex < model()->scanCount(); scanIndex++) {
 		addScan(scanIndex);
 	}
 
+	// Make connections.
+
 	connect(model(), SIGNAL(exclusiveDataSourceChanged(QString)), this, SLOT(onExclusiveDataSourceChanged(QString)));
+	connect( plot_->plot()->signalSource(), SIGNAL(dataPositionChanged(QPointF)), this, SIGNAL(dataPositionChanged(QPointF)) );
 
 	reviewPlotAxesConfiguration(plot_);
 	refreshTitle();
 }
 
 AMScanViewExclusiveView::~AMScanViewExclusiveView() {
-	// PlotSeries's will be deleted as children items of the plot.
-
-	delete plotCursor_;
 	plot_->deleteLater();
-}
-
-void AMScanViewExclusiveView::setPlotCursorVisibility(bool visible)
-{
-	if (visible)
-		plot_->plot()->addItem(plotCursor_);
-
-	else
-		plot_->plot()->removeItem(plotCursor_);
-}
-
-void AMScanViewExclusiveView::setPlotCursorCoordinates(const QPointF &coordinates)
-{
-	plotCursor_->setValue(coordinates);
-}
-
-void AMScanViewExclusiveView::setPlotCursorCoordinates(double xCoordinate)
-{
-	plotCursor_->setValue(QPointF(xCoordinate, plotCursor_->value().y()));
-}
-
-void AMScanViewExclusiveView::setPlotCursorColor(const QColor &color)
-{
-	plotCursor_->marker()->setPen(QPen(color));
-	plotCursor_->marker()->setBrush(QBrush(color));
+	delete dragZoomerTool_;
+	delete wheelZoomerTool_;
+	delete dataPositionTool_;
```

Comment:
I think these will need to be a delete later. Each inherits MPlotAbstractTool which is on the QObject inheritance tree.